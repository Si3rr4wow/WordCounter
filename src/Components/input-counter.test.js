import React from 'react'
import renderer from 'react-test-renderer'
import InputCounter from './input-counter'

it('Renders the text input', () => {
  const tree = renderer
    .create(<InputCounter/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
