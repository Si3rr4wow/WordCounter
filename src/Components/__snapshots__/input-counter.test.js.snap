// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the text input 1`] = `
<div
  className="card"
>
  <div
    className="card-header"
  >
    <h4>
      Use me to analyse smaller bodies of text.
    </h4>
  </div>
  <div
    className="card-body"
  >
    <p>
      I can count the words you type here synchronously
    </p>
    <textarea
      className="form-control"
      onChange={[Function]}
      value=""
    />
  </div>
</div>
`;
